ci/playbooks/edpm_build_images/edpm_build_images_content_provider_run.yaml

```diff
@@ -0,0 +1,48 @@
+---
+- hosts: all
+  gather_facts: true
+  tasks:
+    - name: Build edpm images
+      environment:
+        ANSIBLE_CONFIG: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/ci-framework/ansible.cfg"
+      ansible.builtin.command:
+        chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/ci-framework"
+        cmd: >-
+          ansible-playbook -i localhost,
+          -c local ci/playbooks/edpm_build_images/edpm_build_images_content_provider.yaml
```

Comment:
since you're in the directory, you shouldn't need to pass the "-i localhost, -c local". Also, you shouldn't have to pass the ANSIBLE_CONFIG environment variable.